src/fetcher.ts

```diff
@@ -142,9 +142,10 @@ async function getResponseData(response: Response) {
 async function fetchJson(url: string, init: RequestInit): Promise<ApiResponse> {
   const response = await fetch(url, init)
 
-  const data = await getResponseData(response)
+  const data = await getResponseData(response.clone())
```

Comment:
Will this still fetch the non-blob version even if the we subscribe to the blob stream? I don't see a flag to disable the non-blob fetch.

Reply:
Both the blob and JSON response should be included in the response and either could be used.

Reply:
I thought we wanted to disable the JSON response since it would crash the frontend?

Reply:
Maybe @jpcano can confirm, that was my understanding at least.